### PR TITLE
Code Quality: Replace Vanara with CsWin32

### PR DESCRIPTION
**Pull request by Jack251970**
Removed Vanara.PInvoke.Shell32.CMF & Vanara.PInvokeVanara.PInvoke.RECT import.
Replaced Vanara.PInvoke.Shell32.IApplicationActivationManage with Windows.Win32.UI.Shell.IApplicationActivationManager.
Removed useless Vanara import.

For #15000

### CODE REVIEW

**Review by 0x5bfa — CHANGES_REQUESTED** [review]

File: src/Files.App/Data/Items/ContextMenu.cs [hunk 1]
```diff
@@ -32,6 +32,27 @@ public enum HBITMAP_HMENU : long
 		HBMMENU_SYSTEM = 1
 	}
 
+	// Same definition of Vanara.PInvoke.Shell32.CMF
+	public enum CMF : uint
```
[the remark this answers is not in the document]
Comment (Jack251970) [reply src/Files.App/Data/Items/ContextMenu.cs:36]: It does not seems to work.

File: src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs [hunk 2]
```diff
@@ -77,15 +77,18 @@ public async override Task<List<FileProperty>> LoadPreviewAndDetailsAsync()
 			}
 		}
 
-		public void SizeChanged(RECT size)
+		public void SizeChanged(Windows.Foundation.Rect size)
```
[the remark this answers is not in the document]
Comment (Jack251970) [reply src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs:80]: Well, in the function `Rect GetPreviewSize()` of the same file, `var physicalPos = source.TransformPoint(new Point(0, 0));` uses `Windows.Foundation.Point`. I just keep them in the same namespace.

[the remark this answers is not in the document]
Comment (Jack251970) [reply src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs:80]: Also, `Windows.Win32.Foundation.RECT` has `left`, `right`, `top` and `bottom` properties, which is different from the original `top`, `left`, `width`, `height`.

File: src/Files.App/Data/Items/ContextMenu.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/Data/Items/ContextMenu.cs:36]: I forgot you have to add CFM values we use and use like "PInvoke.CFM_DEFAULT" and pass as an uint type.

[the remark this answers is not in the document]
Comment (Jack251970) [reply src/Files.App/Data/Items/ContextMenu.cs]: Now use the flag from Win32.

File: src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs:80]: Fair, thanks!

File: src/Files.App/Data/Items/ContextMenu.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/Data/Items/ContextMenu.cs]: Thanks!

**Comment by 0x5bfa** [comment]
This looks great, thank you!

While you're at context menu thing, could you replace these manually defined types with cswin32 too?

https://github.com/files-community/Files/blob/08ab49402a6c37e3a36eaccb2738df0b88e6967b/src/Files.App/Data/Items/ContextMenu.cs#L6-L33

**Comment by Jack251970** [comment]
> This looks great, thank you!
> 
> While you're at context menu thing, could you replace these manually defined types with cswin32 too?
> 
> https://github.com/files-community/Files/blob/08ab49402a6c37e3a36eaccb2738df0b88e6967b/src/Files.App/Data/Items/ContextMenu.cs#L6-L33

I will check it later.

**Comment by Jack251970** [comment]
@0x5bfa As for `MFT_SEPARATOR` flag, just write `MFT_SEPARATOR` in NativeMethods.txt cannot help.

As for `HBITMAP_HMENU`, it is only used for checking in `!Enum.IsDefined(typeof(HBITMAP_HMENU), ((IntPtr)menuItemInfo.hbmpItem).ToInt64())`.

**Comment by 0x5bfa** [comment]
Add MENU_ITEM_TYPE instead.

> As for HBITMAP_HMENU, it is only used for checking in !Enum.IsDefined(typeof(HBITMAP_HMENU), ((IntPtr)menuItemInfo.hbmpItem).ToInt64()).

Do you think it can be replaced?

**Comment by Jack251970** [comment]
@0x5bfa Have found `MENU_ITEM_TYPE` and just keep `HBITMAP_HMENU`.

**Review by yaira2 — APPROVED** [review]

**Pull request merged**